apps/meteor/client/views/account/security/ChangePassword.tsx

```diff
@@ -43,6 +43,24 @@ const ChangePassword = (props: AllHTMLAttributes<HTMLFormElement>) => {
 		}
 	};
 
+	function getAriaDescribedbyForPassword(passwordError: boolean) {
+		let str = `${passwordVerifierId}`;
+		if (!allowPasswordChange) {
+			str += ` ${passwordId}-hint ${passwordId}-error`;
+		}
+		if (passwordError) {
+			str += ` ${passwordId}-error`;
+		}
+		return str;
+	}
```

Comment:
I noticed a little issue in this function, with the following properties:
`passwordError = true`
`allowPasswordChange = false`
`passwordId = 'yyy'`
`passwordVerifierId = 'xxx'`
Results in the following string: `'xxx yyy-hint yyy-error yyy-error'`. I think `${passwordId}-error` should not appear twice, correct? Maybe that case doesn't exist, but I wouldn't bet on it, specially since things can change in the future. I suggest the following:

```suggestion
	function getAriaDescribedbyForPassword(
		passwordVerifierId: string,
		passwordId: string,
		allowPasswordChange: boolean,
		passwordError: boolean,
	) {
		const error = !allowPasswordChange || passwordError;
		return [passwordVerifierId, !allowPasswordChange && `${passwordId}-hint`, error && `${passwordId}-error`].filter(Boolean).join(' ');
	}
```

Then we can remove the function declaration from the Component's body (just define the function above the component). This is not a big deal but since this function will run every time anyways theres no reason for us to not do this, as it helps with readability and separation of concerns.

Reply:
ok